examples/with-aws-serverless/src/pages/_app.tsx

```diff
@@ -0,0 +1,35 @@
+import { NextComponentType, NextPageContext } from 'next'
+import App from 'next/app'
+import { Router } from 'next/dist/client/router'
+import React from 'react'
+import GNB from 'src/components/GNB'
+
+type AppProps = {
+  Component: NextComponentType<NextPageContext>
+  router: Router
+  pageProps: any
+  err: any
+}
+
+export class CustomApp extends App<AppProps> {
```

Comment:
Custom _app is generally written as: https://nextjs.org/docs/advanced-features/custom-app